Feat - Slack app to get URL's

Jocapear opened the pull request:
#### What does this PR do?

Add scripts that get domains from urls in messages from a slack channel.
A module allows request from the slack api to get messages from all
channels, a scripts parses the information and creates a JSON with all
the channels and the count of domains mentioned in the messages, the
domains are grouped by day. Also added a text document that contains the
names of all the required python libraries.

#### Where should the reviewer start?

Read the URLGetter.py script and follow the code from there.

#### How should this be manually tested?
1.- Get a slack api token and put in the URLGetter.py script
2.- Run URLGetter.py script `python URLGetter.py`
3.- Check the JSON file that was created

#### Any background context you want to provide?
This feature is to get information about what web pages are mentioned more in a slack workspace or channel. This information is going to be used to focus the webpage parsing of the chrome extension in the most mentioned web pages.

#### What are the relevant tickets?

Fixes #4 

#### Screenshots
Not available

#### Questions
Is there a better way to parse the messages into a JSON file?
Does it need to handle errors? Remember that we are making API requests
Does it need more testing?

#### Checklist
- [X] I added the necessary documentation, if appropriate.
- [X] I added tests to prove that my fix is effective or my feature works.
- [X] I reviewed existing Pull Requests before submitting mine.

Jocapear committed:
Add scripts that get domains from urls in messages from a slack channel.

A module allows request from the slack api to get messages from all
channels, a scripts parses the information and creates a JSON with all
the channels and the count of domains mentioned in the messages, the
domains are grouped by day. Also added a text document that contains the
names of all the required python libraries.

marconi1992 commented on slack_app/URLGetter.py:
```diff
@@ -0,0 +1,51 @@
+    writeJSON(jsonName, result)
+    
+today = str(datetime.now()).split()[0]
+slack_api_key = API_KEY_HERE
```
We could get the API key using an environment variable.

```python
import os

slack_api_key = os.environ.get('SLACK_API_KEY')
```

Jocapear replied:
Done. Be sure to tell me anything else about best practices or errors. 😃 

https://github.com/wizelineacademy/saga-chrome/pull/5/commits/d968c0df9ce47281b9f1de611b3be76a167bc068

Jocapear committed:
Change api key retreival method

The slack api key now is retreived from an environmental variable called
SLACK_API_KEY. If the variable is not send, prints a message with an
error and tells the user to set the variable.
